### ISSUE TITLE
npm ci stop(emojione)

### ISSUE THREAD

**Issue by jbshin-gemiso** [albirrkarim/mozilla-hubs-installation-detailed#14]
hello.

Thanks for making the mozilla hubs installation tutorial.
I ran into a problem during installation following your guide.
I've searched and searched for the cause, but I can't find a solution.
I want to get help.

1.4 Hubs
Clone and install dependencies in progress
npm ci hangs while running.
No matter how many times I try, the same emojione freezes.

I attach a screenshot below.

![2022-07-15 112926](https://user-images.githubusercontent.com/75593521/179134946-2f8a655e-4143-4fa7-bdae-98a3bd0a1878.png)

After npm ci failed, I tried npm install , but it still fails. in the same section.

The failure section is attached to the screenshot above.

I am using WSL2 on Windows 10 operating system.
I am using Ubuntu 20.04.4 LTS version.

Nodejs version is 10.19.0.
elixir version is 1.12.3-otp-23 .
erlang version is 23.2.1 .

Everything else was installed according to the procedure you gave.

Any help would be appreciated.
have a good day.

**Comment by albirrkarim** [comment]
How long it stuck on emojione ?  in minute / hour ?

tell me your hardware (CPU& RAM) and your internet speed

**Comment by jbshin-gemiso** [comment]
I'm stuck on emojione forever.
I am in a situation where I have to abort the installation.

My PC specs requested are
CPU: intel i7-7700 3.60ghz
RAM: 64GB
internet : 100Mbps

**Comment by jbshin-gemiso** [comment]
The problem seems to have been resolved.

I updated the versions of 'npm' and 'nodejs', downloaded hubs.git again, and installed it.
Another difference is, when the error occurred before, I was using linux installed on wsl2 using 'windows terminal'.

But I didn't use windows terminal this time. (using wsl2-ubuntu launcher directly)
I believe the latter would not have had much of an impact, and I believe the former solved the problem.
